### PR TITLE
xeyes update to 1.1.2

### PR DESCRIPTION
**Pull request by alarcher**

### CODE REVIEW

File: make-rules/x11.mk [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+# Define library install paths
+#
+
+XORG_LIBDIR.32 = $(USRLIBDIR)/xorg	
+XORG_LIBDIR.64 = $(USRLIBDIR)/xorg/$(MACH64)	
+XORG_LIBDIR    = $(XORG_LIBDIR.$(BITS))
+
+#
+# Common macros for font packages
+#
```
Comment (pyhalov) [inline make-rules/x11.mk]: Sorry, I don't see their usage in our xorg components. Do we really need them now?

File: make-rules/x11.mk [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+# CDDL HEADER START
```
Comment (pyhalov) [inline make-rules/x11.mk]: Can you use new Makefile header?

File: make-rules/x11.mk (hunk 1 above)
Comment (alarcher) [reply make-rules/x11.mk]: The corresponding component is in another branch. I can split if you like.

File: make-rules/x11.mk (hunk 2 above)
Comment (alarcher) [reply make-rules/x11.mk]: OK

**Pull request merged**